whois-query/src/main/java/net/ripe/db/whois/query/hazelcast/HazelcastInstanceManager.java

```diff
@@ -14,42 +14,67 @@
 import org.springframework.context.annotation.Primary;
 import org.springframework.context.annotation.Profile;
 
+import java.util.Arrays;
+
 @Configuration
 @DeployedProfile
 public class HazelcastInstanceManager {
     private static final Logger LOGGER = LoggerFactory.getLogger(HazelcastInstanceManager.class);
 
     final String interfaces;
+    final int port;
 
     @Autowired
-    public HazelcastInstanceManager( @Value("${hazelcast.config.interfaces}") final String interfaces) {
+    public HazelcastInstanceManager(@Value("${hazelcast.config.interfaces}") final String interfaces, @Value("${hazelcast.port:5701}") final int port) {
```

Comment:
It's not part of this PR (hazelcast.config.interfaces), but you can use Spring to split a comma-separated value for you, and inject into a list. For example, `@Value("#{'${email.excluded:}'.split(',')}")` in `ExcludedEmailValidator.java`

Reply:
I did not used it specifically as it is also used in aws one and there it won't be a list. As addInterface does not take list which we use in aws but for tcp config it has that function to add list of members

Reply:
OK! Thanks